### ISSUE TITLE
streaming api not working 

### ISSUE THREAD

**Issue by nchamroukhi** [oobabooga/text-generation-webui#4591]
### Describe the bug

after latest updates when launch the app using the docker image thebloke/cuda11.8.0-ubuntu22.04-oneclick:latest 

7860 UI work fine and i can load the model
5005 for streaming api not work, when before it work fine for me

### Is there an existing issue for this?

- [x] I have searched the existing issues

### Reproduction

just run the image and try to access to 5005 port for streaming api 

### Logs

```shell
no logs
```

### System Info

```shell
cannot connect to URI ip:5005
```

**Comment by leomie** [comment]
The api has been changed in favor to openAI api, you would need to use older version that supports the old API, or refer to this [repo](https://github.com/ashleykleynhans/oobabooga-legacy-api-extension)

**Comment by nchamroukhi** [comment]
did you have any documentation to update our code to use the openAI api ?

**Comment by leomie** [comment]
You can find some useful documentation [here](https://github.com/oobabooga/text-generation-webui/blob/main/docs/12%20-%20OpenAI%20API.md) thats all we got at the moment. Personally, I will avoid the openAI api as we have seen reports of bugs which have not been sorted yet, it also seem to be lacking some features.

**Comment by sam1am** [comment]
Gotta say... removing a production api endpoint from an auto-updating app without notice is a pretty bad move. How many people's days must be ruined?

**Comment by Morphus1** [comment]
Yup, had a nice little setup using history for RAG....

**Comment by emersonium** [comment]
om my gosh, so sad I upgraded :'()

**Comment by emersonium** [comment]
> Gotta say... removing a production api endpoint from an auto-updating app without notice is a pretty bad move. How many people's days must be ruined?

Agreed, this makes for a rough weekend, gota figure out what this openai business is all aboot
